### PR TITLE
Fix integrity when updating from 9.1.x to master

### PR DESCRIPTION
**Pull request by VicDeo**
Fixing issue reported at https://github.com/owncloud/administration/pull/111#issuecomment-283978146

extra files are left after update from  9.1.x:
```
Results
=======
- core
	- EXTRA_FILE
		- updater/nbproject/project.xml
		- updater/nbproject/project.properties
```

### CODE REVIEW

File: src/Tests/bootstrap.php [hunk 1]
```diff
@@ -6,5 +6,6 @@
 require_once __DIR__.'/../../vendor/autoload.php';
 
 if(!class_exists('PHPUnit_Framework_TestCase')) {
-	require_once('PHPUnit/Autoload.php');
+	class PHPUnit_Framework_TestCase extends \PHPUnit\Framework\TestCase {
```
Comment (DeepDiver1975) [inline src/Tests/bootstrap.php]: no - again!

**Comment by davitol** [comment]
@VicDeo Due to https://github.com/owncloud/updater/pull/417 is a fix for master, in order to test this PR, the only way I can do it is "hacking" the update server config.php with an url with a master version patched with the changes located in my local server. Am I right?

**Comment by VicDeo** [comment]
@davitol ah, indeed. It needs to applied on top on stable9.1

**Comment by VicDeo** [comment]
@davitol 
 ###Steps
1. Install daily build
2. create nbproject https://github.com/owncloud/updater/pull/412/files
3. Apply one-liner https://github.com/owncloud/updater/pull/417/files#diff-144577de21853edd589e20353ca4305b
4. update to daily

Result
With step 3: Intergrity  check :green_heart: 
Without step 3: Intergrity  check :boom:  

Needs to be backported to the next 9.1

**Comment by davitol** [comment]
:+1: 
It solves the integration check problem of nbproject

**Pull request merged**